### PR TITLE
Fixes MahaanKoshTooltip Reading mode bug: #1741 + Prettier formatting 

### PR DESCRIPTION
**Pull request by JapnitS**
* [x]  Passed [sanity tests](http://bit.ly/sttm-sanity-tests).
* [x]  Ran `npm test` & fixed newly introduced lint errors.
* [x]  Checked console for errors.

Fixes bug #1741 by wrapping `BaaniLine` in `MahaanKoshContext.provider`.
+ Main logic diff [pointer](https://github.com/KhalisFoundation/sttm-web/compare/dev...JapnitS:sttm-web:Bugfix-MahaanKoshTooltipReadingMode#diff-e317174e052c1908e085efa0da22d1ed1fb5a32fc7f89246c30d0487c48c9cc7R888-R916)

+ Test snapshot change
+ Prettier formatting
<!-- New to markdown? Simply put an 'x' between [ ] to check it. Like [x] this. (No spaces).

### CODE REVIEW

**Comment by gurjit03** [comment]
> * [x]   Passed [sanity tests](http://bit.ly/sttm-sanity-tests).
> * [x]   Ran `npm test` & fixed newly introduced lint errors.
> * [x]   Checked console for errors.
> 
> Fixes bug #1741 by wrapping `BaaniLine` in `MahaanKoshContext.provider`.
> 
> * Main logic diff [pointer](https://github.com/KhalisFoundation/sttm-web/compare/dev...JapnitS:sttm-web:Bugfix-MahaanKoshTooltipReadingMode#diff-e317174e052c1908e085efa0da22d1ed1fb5a32fc7f89246c30d0487c48c9cc7R888-R916)
> * Test snapshot change
> * Prettier formatting

@JapnitS Great work 🔥  fixing this bug. Can we kindly remove this prettier formatting as we mostly relying on the VScode internal formatter so that we have similar formatting across the product.

Rather I would propose to create another ticket to move the formatting to the prettier and we can do so in 1 formatting PR.
cc @Gauravjeetsingh

File: src/js/components/Baani/Baani.js [hunk 1]
```diff
@@ -495,154 +498,154 @@ class Baani extends React.PureComponent {
...
-                  ))}
-                </div>
-                {/* Hindi Translation Languages */}
-                <div
-                  className={`${mixedViewBaaniClass}-translation ${paragraphModeClass}`}
-                >
-                  {hindiTranslationLanguages.map((source) => (
+                  <>
                     <div
-                      key={source}
```
Comment (gurjit03) [inline src/js/components/Baani/Baani.js:622]: @JapnitS here we are fixing the key to the spanish language

File: src/js/components/Baani/Baani.js [hunk 2]
```diff
@@ -495,154 +498,154 @@ class Baani extends React.PureComponent {
...
                     </div>
-                  ))}
-                </div>
-                
-                {isParagraphMode ? null : (
+                  </>
+                )}
+              </div>
+              {/* Punjabi Translation Languages */}
+              <div
```
Comment (gurjit03) [inline src/js/components/Baani/Baani.js:602]: @JapnitS Lets create a small component named DisplayMixedViewBaani and use that here.

**Review by gurjit03 — CHANGES_REQUESTED** [review]

**Comment by JapnitS** [comment]
Closed in favor of https://github.com/KhalisFoundation/sttm-web/pull/1743

**Pull request closed without merging**